backend/src/main/kotlin/com/ksidelta/libruch/modules/borrowing/BorrowingAggregate.kt

```diff
@@ -0,0 +1,79 @@
+package com.ksidelta.libruch.modules.borrowing
+
+import com.ksidelta.libruch.modules.kernel.Party
+import org.axonframework.commandhandling.CommandHandler
+import org.axonframework.eventsourcing.EventSourcingHandler
+import org.axonframework.extensions.kotlin.applyEvent
+import org.axonframework.modelling.command.AggregateIdentifier
+import org.axonframework.modelling.command.TargetAggregateIdentifier
+import org.axonframework.spring.stereotype.Aggregate
+import java.util.*
+
+/**
+ * After you have successfully added Product
+ * in the ProductAggregate, then an event should be sent, which will create a borrowing offer
+ * which effectively, makes the book available for exchange
+ */
+@Aggregate
+class BorrowingAggregate() {
+
+    @AggregateIdentifier
+    lateinit var borrowing: UUID
+
+    @CommandHandler
+    constructor(command: RegisterNewBorrowing): this() {
+
+    }
+
+    @CommandHandler
+    fun findBorrowings(command: FindBorrowings){
```

Comment:
It will be done by queries on read model

Reply:
done